index.html

```diff
@@ -3,17 +3,127 @@
+					<a class="nav-link" href="recipes.html">Recipes</a>
+				</li>
+			</ul>
+			<div class="collapse navbar-collapse" id="navbarNavDarkDropdown">
+				<ul class="navbar-nav">
+					<li class="nav-item dropdown">
+						<a class="nav-link dropdown-toggle" href="#" id="navbarDarkDropdownMenuLink" role="button"
+							data-bs-toggle="dropdown" aria-expanded="false">
+							Menu
+						</a>
+						<ul class="dropdown-menu dropdown-menu-dark" aria-labelledby="navbarDarkDropdownMenuLink">
+							<li><a class="dropdown-item" href="#">Hot out of the Oven</a></li>
+							<li><a class="dropdown-item" href="#">Specialty Pizzas</a></li>
+						</ul>
+					</li>
+				</ul>
+			</div>
 		</div>
-</body>
+	</nav>
+	<h1>Our Story</h1>
+	<p>The Pizza Party story is really the story about America, from Naples to New York City to Buffalo, NY. In 1938 at
+		the age of 12 and 15 Giovanni and Carlo Pardini left their home in Naples, Italy to achieve the American dream.
+		<br>
+	<p>After a short stay at Ellis Island. Their name was shortened to <i>"Pardi"</i>, they lived in New York City until
+		leaving for Buffalo in 1950. The brothers opened <em>"Pizzeria di Pardi"</em> on Grant street in Buffalo's west
+		side. In 1986 we expanded to 8 locations in western NY and Pizzeria di Pardi became <b>Pizza Party! </b> Today
+		we have three locations serving up the classic pizza & wings Buffalo is known for our original location as well
+		as our Hertel Avenue and UB locations.</p>
+
+	<h1>Menu</h1>
+	<h2>HOT out of the Oven</h2>
+	<table>
+		<thead>
+			<th>Fresh Baked Pizza</th>
+			<th>Personal-12"</th>
+			<th>Large-18"</th>
+		</thead>
+		<tbody>
+			<tr>
+				<td>Cheese</td>
+				<td>$12.00</td>
+				<td>$18.00</td>
+			</tr>
+			<tr>
+				<td>Cheese with One Topping</td>
+				<td>$14.00</td>
+				<td>$20.00</td>
+			</tr>
+			<tr>
+				<td>Extra Cheese</td>
+				<td>$3.00</td>
+				<td>$4.00</td>
+			</tr>
+			<tr>
+				<td>Beef it up</td>
+				<td>$4.00</td>
+				<td>$6.00</td>
+			</tr>
+		</tbody>
+	</table>
+
+	<div>
+		<p><b>Choice of Toppings:</b> Black Olives, Broccoli, Fresh Onions, Green Olives, Mushrooms, Italian Sausage,
+			Buffalo Chicken, Pepperoni, Ham.</p>
+	</div>
+	<hr>
+
+	<div>
+		<h3><b>Specialty Pizzas</b></h3>
+		<hr>
+
+		<h4><b>Flyers</b>$16.50</h4>
+		<p>Slices of pepperoni pizza are called flyers, because of the way they are often tossed around like Frisbees
+			with Fresh mozzarella and Fresh Basil.</p>
+		<hr>
+
+		<h4><b>Grandma's Pizza</b>$12.50</h4>
+		<p>Grandma pizza is a distinct pizza that originates from Long Island, New York. It is a thin, square pizza with
+			Cheese, Tomatoes, and Fresh Basil</p>
+		<hr>
 
+		<h4><b>Gmoney</b>$14.00</h4>
+		<p>Queens style pizza with Extra Cheese and side of Marinara sauce.</p>
+		<hr>
+
+		<h4><b>Becca</b>$25.00</h4>
+		<p>I love when it's Hot Bread in the oven</p>
+		<hr>
+
+		<h4><b>Chris</b>$20.00</h4>
+		<p>Pepperoni, Bacon & Peppers.</p>
+		<hr>
+
+		<h1>Recipe</h1>
+		
+		</div>
```

Comment:
each of these menu items should be in their own container. possibly a bootstrap card or something of that nature